install_sct

```diff
@@ -527,6 +543,12 @@ yes | python/bin/conda create -n venv_sct python=3.6
 # activate miniconda
 source python/etc/profile.d/conda.sh
 conda activate venv_sct
+# double-check conda activated (it can be glitchy): https://github.com/neuropoly/spinalcordtoolbox/issues/3029
+if [ "$(realpath "$(command -v python)")" != "$(realpath "$(command -v python/envs/venv_sct/bin/python)")" ]; then
+  echo "error: venv_sct failed to activate. Do you have a conflicting python installation?" >&2
```

Comment:
I think `Do you have a conflicting python installation?` might not be informative enough. Perhaps we could give the value of `$(realpath "$(command -v python)` in the error message?  

```suggestion
# double-check conda activated (it can be glitchy): https://github.com/neuropoly/spinalcordtoolbox/issues/3029
export EXPECTED_PYTHON = "$(realpath "$(command -v python/envs/venv_sct/bin/python)")"
export ACTUAL_PYTHON = "$(realpath "$(command -v python)")"
if [ "$ACTUAL_PYTHON" != "$EXPECTED_PYTHON"  ]; then
  echo "Error: Activating venv_sct failed to switch Python interpreter to Miniconda. (Incorrect interpreter used: $ACTUAL_PYTHON)" >&2
```

Reply:
Side question about shell scripts: Would it be necessary to use `export` here in my suggestion? (See also https://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export)

Reply:
`export` means that subprocesses receive it in their `os.environ` (or equivalent for whatever lang you're working in). People think `export` means `set` but it doesn't, the `=` means `set`. Also, you can set env for a specific subprocess like this:

```
DEBIAN_FRONTEND=noninteractive SOMEOTHERVAR=somethingelse apt-get install -y
```

That's the same as quoting like:

```

```
(
  export DEBIAN_FRONTEND=noninteractive
  export SOMEOTHERVAR=somethingelse
  apt-get install -y
)
```

In general I'd only use `export` if I really wanted to set something globally, like if I was running `apt` multiple times in a script or needed to change where `TMPDIR` was.

Reply:
Whiiiich is all to say that I'll use this instead:

(another shell gotcha: you can't have spaces around `=`!!)

```suggestion
# double-check conda activated (it can be glitchy): https://github.com/neuropoly/spinalcordtoolbox/issues/3029
EXPECTED_PYTHON="$(realpath "$(command -v python/envs/venv_sct/bin/python)")"
ACTUAL_PYTHON="$(realpath "$(command -v python)")"
if [ "$ACTUAL_PYTHON" != "$EXPECTED_PYTHON"  ]; then
  echo "Error: Activating venv_sct failed to switch Python interpreter to Miniconda. (Incorrect interpreter used: $ACTUAL_PYTHON)" >&2
```